docs/customize-question-types-third-party-component-integration-react.md

```diff
@@ -244,14 +251,66 @@ PropertyGridEditorCollection.register({
     return prop.type === "color";
   },
   getJSON: function () {
-    return { type: CUSTOM_TYPE };
+    return {
+      type: CUSTOM_TYPE,
+      colorPickerType: "Compact"
+    };
   }
 });
 ```
 
-[View Full Code on GitHub](https://github.com/surveyjs/code-examples/tree/main/integrate-third-party-react-components  (linkStyle))
+To try the functionality, you can add a custom property of the `"color"` type to the survey. The code below adds a custom `backgroundColor` property. When users change its value, they change the `--background` CSS variable. The [`onActiveTabChanged`](https://surveyjs.io/survey-creator/documentation/api-reference/survey-creator#onActiveTabChanged) event handler reapplies the selected background color when users switch to the Designer or Preview tab.
+
+```js
+import { SurveyCreator, SurveyCreatorComponent } from "survey-creator-react";
+import { Serializer } from "survey-core";
+import { registerColorPicker } from "./ColorPicker";
+import "survey-core/defaultV2.css";
+import "survey-creator-core/survey-creator-core.css";
+
+registerColorPicker();
+addBackgroundColorProperty();
+
+export function SurveyCreatorWidget () {
+  const creator = new SurveyCreator();
+  creator.onActiveTabChanged.add(handleActiveTabChange);
+  return <SurveyCreatorComponent creator={creator} />;
+}
+
+function addBackgroundColorProperty() {
+  Serializer.addProperty("survey", {
+    name: "backgroundColor",
+    displayName: "Background color",
+    type: "color",
+    category: "general",
+    visibleIndex: 3,
+    onSetValue: (survey, value) => {
+      survey.setPropertyValue("backgroundColor", value);
+      applyBackground(value);
+    }
+  });
+}
+
+function applyBackground(color) {
+  setTimeout(() => {
+    const surveyEl = document.getElementsByClassName("sd-root-modern")[0];
+    if (!!surveyEl) {
+      surveyEl.style.setProperty("--background", color);
+    }
+  }, 50);
+}
+
+function handleActiveTabChange(sender, options) {
+  if (options.tabName === "test" || options.tabName === "designer") {
+    applyBackground(sender.survey.backgroundColor);
+  }
+}
+```
+
+[View Demo](/survey-creator/examples/custom-colorpicker-property-editor/react)
+[View Full Code on GitHub](https://github.com/surveyjs/code-examples/tree/main/integrate-third-party-react-components (linkStyle))
 
-## See also
+## See Also
 
 The following demo shows how to use the [React Select](https://react-select.com/home) control as a drop-down editor for a survey question.
```

Comment:
Refer to the following article for more information on customizing a predefined theme using CSS variables: [Customize Themes](https://surveyjs.io/form-library/documentation/manage-default-themes-and-styles).